js/drop.js

```diff
@@ -29,6 +29,18 @@ export const DropAPI = superclass =>
             const drop = await this.delete(url);
             return drop;
         }
+
+        async addProduct(id, payload) {

```

Comment:
Because this is used as a mixin it should probably have a suffix to avoid future collisions. We need to follow the strict naming policy we're using for nested resources in other places (https://github.com/ripe-tech/ripe-registry-api-js/blob/master/js/client.js). In this case, I believe it would be `createProductDropDrop`, right?

Reply:
Yes (https://github.com/ripe-tech/ripe-registry-api-js/blob/master/js/client.js#L97) but we are cutting the "Drop" from "ProductDrop" whenever the context is the Drop. So, `createProductDrop` it's a product for a drop, `createProductCollection` is a product for a collection.